docs/modules/hive/pages/index.adoc

```diff
@@ -1,11 +1,12 @@
 = Stackable Operator for Apache Hive
 
 This is an operator for Kubernetes that can manage https://hive.apache.org[Apache Hive] metastores.
+The Apache Hive metastore (HMS) stores information on the location of tables and partitions in file and blob storages such as HDFS and S3.
 
-Currently it only supports running a Hive metastore service (HMS) that stores metadata about schemas and tables.
-
-Due to several reasons we do not feel that running Hive on Kubernetes makes much sense these days. The most obvious reason being that Hive requires YARN as execution framework, which basically tries to do the same job as Kubernetes - i.e. assign resources. 
-For this reason we decided to provide [Trino](https://github.com/stackabletech/trino-operator) as query engine in the Stackable Data Platform instead of Hive - which still uses the Hive Metastore, hence this operator exists.
+Only the metastore is supported, not Hive itself.
+Due to several reasons we do not feel that running Hive on Kubernetes makes much sense these days.
+The most obvious reason being that Hive requires YARN as execution framework, which basically tries to do the same job as Kubernetes - i.e. assign resources. 
+For this reason we decided to provide xref:trino:index.adoc[Trino] as a query engine in the Stackable Data Platform instead of Hive - which still uses the Hive Metastore, hence this operator exists.
```

Comment:
```suggestion
For this reason we provide xref:trino:index.adoc[Trino] as a query engine in the Stackable Data Platform instead of Hive. Trino still uses the Hive Metastore, hence the inclusion of this operator as well.
```